### PR TITLE
Add HEC{R,H} to wcs_to_ivoa mapping

### PR DESCRIPTION
**Pull request by dstansby**
Fixes https://github.com/sunpy/ndcube/issues/366. I think "Body related coordinate (altitude on the body)" is correct for both of these; one is measured from the center of the Sun, and one is measured from the solar surface according to Thompson 2006.

### CODE REVIEW

**Comment by DanRyanIrish** [comment]
Hi @dstansby.  Thanks so much for this PR.  I think this change should actually be made in astropy, here: https://github.com/astropy/astropy/blob/master/astropy/wcs/wcsapi/fitswcs.py#L88

Also, if these are different CTYPES giving different, if similar, properties, then they should ideally have different IVOA strings.  If needs be we can create one and prepend it with `custom:`.

File: ndcube/utils/wcs.py [hunk 1]
```diff
@@ -44,7 +44,9 @@ def inv(self):
     "PIXEL": "instr.pixel",
     "XPIXEL": "custom:instr.pixel.x",
     "YPIXEL": "custom:instr.pixel.y",
-    "ZPIXEL": "custom:instr.pixel.z"
+    "ZPIXEL": "custom:instr.pixel.z",
+    "HECR": "pos.bodyrc.alt",
```
Comment (DanRyanIrish) [inline ndcube/utils/wcs.py:48]:
```suggestion
    "HECR": "custom:pos.heliographic.distance",
```

Comment (DanRyanIrish) [reply ndcube/utils/wcs.py:48]: I think `"HECR": "pos.bodyrc.alt"` makes sense but have suggested a change for `"HECR"`

**Comment by dstansby** [comment]
> Hi @dstansby. Thanks so much for this PR. I think this change should actually be made in astropy, here: https://github.com/astropy/astropy/blob/master/astropy/wcs/wcsapi/fitswcs.py#L88

Is there something special about the ones currently in `ndcube` that means they shouldn't be upstream in astropy?

**Comment by Cadair** [comment]
A lot of them are now, we are just waiting for release of astropy 4.3 to include them. We should actually remove some of these from this list now as they are in 4.2.

**Review by Cadair — APPROVED** [review]

**Pull request merged**